buildkite/scripts/dhall/dump_dhall_to_pipelines.sh

```diff
@@ -11,13 +19,16 @@ echo "Dumping pipelines from '$ROOT' to '$OUTPUT'"
 
 COUNTER=0
 
-for file in "$ROOT"/**/*.dhall
+for file in "$ROOT"/Jobs/**/*.dhall
 do
     filename=$(basename "$file")
     filename="${filename%.*}"
 
     dhall-to-yaml --quoted --file "$file" > "$OUTPUT"/"$filename".yml
 
+    dhall-to-yaml <<< "let SelectFiles = $ROOT/Lib/SelectFiles.dhall in SelectFiles.compile (./$file).spec.dirtyWhen " > "$OUTPUT"/"$filename".dirtywhen
```

Comment:
Dumping .dirtywhen files as dhall compiles them entirely and we don't want to repeat the same logic here yet